### PR TITLE
Add 'includeChanged' option

### PR DESCRIPTION
**Pull request by vitalbone**
This PR adds the ability to export entries with the `inlcudeChanged` status.

### CODE REVIEW

File: lib/tasks/get-space-data.js [hunk 1]
```diff
@@ -228,3 +231,7 @@ function filterDrafts (items, includeDrafts, cdaClient) {
 function filterArchived (items, includeArchived) {
   return includeArchived ? items : items.filter((item) => !item.sys.archivedVersion)
 }
+
+function filterChanged (items, includeChanged) {
+  return includeChanged ? items : items.filter((item) => !item.sys.changedVersion)
```
Comment (vitalbone) [inline lib/tasks/get-space-data.js:236]: TBH I don't know if `changedVersion` is a thing but I sure hope so 😄

**Comment by vitalbone** [comment]
@andreascful @marcolink 👋 hello, do you think you'd be able to take a look at this please? Positive or ~negative~ not-so-positive feedback encouraged. We'd love to get this functionality in (whether its via this PR or some other way)

**Comment by andreascful** [comment]
Hello Vito,

thank you for your PR.
I had a look at it and have a question. Could you clarify exactly what you are trying to achieve with the feature? What is the intended behavior compared to the behavior without having this flag/option?

Furthermore I have to add that we unfortunately have no "changedVersion" property in our response. Therefore we'll have to think about another way to do this. 

Best wishes
Andreas

**Comment by vitalbone** [comment]
Hi @andreascful thanks for responding. The original intention was to export entries with the status of "changed". However since you asked your question I've double-checked and it looks as though I might have been incorrect. Are you able to confirm? Thanks

**Comment by andreascful** [comment]
When looking at the PR it seems like it is trying to achieve getting only changed entries. However, since the changedVersion field does not exist it is a little ambiguous and open to interpretation.

On a different note, would the usage of the [Preview API ](https://www.contentful.com/developers/docs/references/content-preview-api/)fulfil what you are trying to achieve with this PR?

**Pull request closed without merging**